src/test/java/org/jenkinsci/remoting/protocol/ProtocolStackImplTest.java

```diff
@@ -447,7 +448,6 @@ public void socketBasicBackToBackWithAckSSLEngineHeaders() throws Exception {
     }
 
     @Test
-    @Repeat(16)
```

Comment:
Otherwise can get timeouts. Anyway, why did we need to run these 16 times to “prove” that the code was correct?